Source/WebCore/platform/graphics/gbm/DMABufFormat.h

```diff
@@ -126,6 +126,12 @@ struct DMABufFormat {
             , verticalSubsampling(PlaneDefinitionType::verticalSubsampling)
         { }
 
+        Plane(const FourCC& fourcc, const unsigned& hsValue, const unsigned& vsValue)
```

Comment:
The constructor used during deserialization could be made `private` by adding a `friend` statement like this:

```cpp
friend struct IPC::ArgumentCoder<Planem, void>;
```

as part of the `struct Plane` declaration.